Formula/kubernetes-service-catalog-client.rb

```diff
@@ -0,0 +1,33 @@
+class KubernetesServiceCatalogClient < Formula
+  desc "Consume Services in k8s using the OSB API"
+  homepage "https://svc-cat.io/"
+  url "https://github.com/kubernetes-incubator/service-catalog.git",
+      :tag => "v0.1.23",
+      :revision => "87a5db0e1e0359ce372037a235ce4448944c8611"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["NO_DOCKER"] = "1"
+    dir = buildpath/"src/github.com/kubernetes-incubator/service-catalog"
+
```

Comment:
Move `dir = buildpath/"src/github.com/kubernetes-incubator/service-catalog"` down one line & `dir.install buildpath.children` up one line. This is nit-picking but since other changes are being requested figured it's hopefully not too annoying to ask.

Reply:
Updated it. Code readability is important. Thanks for being thorough.